Homework 11 and 12

KatrinKancheva opened the pull request:
Жени, изпращам и двете домашни в един файл, като това е едно от най-трудните домашни, които сме имали според мен :+1: :)

zzeni commented on ivo/hw16/3.1/3.1.js:
```diff
@@ -0,0 +1,6 @@
+setTimeout(url, 30000);
```
браво за тази задача :)

zzeni commented on ivo/hw16/3.2/3.2.js:
```diff
@@ -0,0 +1,15 @@
+}
+
+function myStopFunction() {
+    clearInterval(myVar);
```
супер е. Само дето са ти в глобалният контекст всички функции, но пак е ок ;)

zzeni commented on ivo/hw16/1/1.js:
```diff
@@ -0,0 +1,48 @@
+];
+
+function describe(car) {
+    if (car.color === "red") {
```
филтрирането по цвят е подобно. Просто трябва да напишеш филтриращата функция ето така:

```
var redCars = cars.filter(function(car) {
  return car.color === "red";
});
```

KatrinKancheva committed:
Homework 11 and 12

Жени, изпращам и двете домашни в един файл, като това е едно от най-трудните домашни, които сме имали според мен :+1: :)

zzeni commented:
Справила си се чудесно!
Било ти е трудно защото Bootstrap ти е непознат все още. След като свикнеш с него , ще ти е много по лесно вече ;)
